base/ca/src/main/java/org/dogtagpki/server/ca/rest/v2/KRAConnectorServlet.java

```diff
@@ -0,0 +1,113 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package org.dogtagpki.server.ca.rest.v2;
+
+import java.io.PrintWriter;
+import java.util.stream.Collectors;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.netscape.certsrv.base.EBaseException;
+import com.netscape.certsrv.base.PKIException;
+import com.netscape.certsrv.base.WebAction;
+import com.netscape.certsrv.system.KRAConnectorInfo;
+import com.netscape.certsrv.util.JSONSerializer;
+import com.netscape.cms.servlet.admin.KRAConnectorProcessor;
+
+/**
+ * @author Marco Fargetta {@literal <mfargett@redhat.com>}
+ * @author Ade Lee
+ */
+@WebServlet(
+        name = "caKraConnector",
+        urlPatterns = "/v2/admin/kraconnector/*")
```

Comment:
For consistency we probably should use plurals (i.e. `kraconnectors`), but we can do it separately later. I think this is one of the earliest REST services we had so the naming convention wasn't really established.

Reply:
Yes, we can review all paths and operations when all services are converted